code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/IdentityConstants.java

```diff
@@ -15,6 +15,7 @@ final class IdentityConstants {
 
 	static final String LOG_TAG = "EdgeIdentity";
 	static final String EXTENSION_NAME = "com.adobe.edge.identity";
+	static final String EXTENSION_FRIENDLY_NAME = "Edge Identity";
```

Comment:
can we call it Identity for Edge Network for consistency with docs and Data Collection card description?

Reply:
Is this change planned for [iOS](https://github.com/adobe/aepsdk-edgeidentity-ios/blob/main/Sources/IdentityConstants.swift#L18) as well? Want to keep is consistent whichever way.

Reply:
good point, let's keep them consistent then "Edge Identity"